### PR TITLE
Added tearDownClass to tests contains setUpClass

### PR DESCRIPTION
**Pull request by ar4s**
Flush the database in tearDownClass.

### CODE REVIEW

File: src/ralph_assets/tests/unit/tests_other.py [hunk 1]
```diff
@@ -88,6 +88,11 @@ def setUpClass(cls):
         )
         cls.licence1.save()
 
+    @classmethod
+    def tearDownClass(cls):
+        from django.core.management import call_command
```
Comment (xliiv) [inline src/ralph_assets/tests/unit/tests_other.py]: import to top of file, possible?

File: src/ralph_assets/tests/unit/tests_other.py [hunk 2]
```diff
@@ -208,6 +213,11 @@ def setUpClass(cls):
         cls.asset1 = BOAssetFactory()
         cls.asset2 = BOAssetFactory()
 
+    @classmethod
+    def tearDownClass(cls):
+        from django.core.management import call_command
```
Comment (xliiv) [inline src/ralph_assets/tests/unit/tests_other.py]: import to top of file, possible?

File: src/ralph_assets/tests/unit/tests_other.py [hunk 3]
```diff
@@ -303,6 +313,11 @@ def setUpClass(cls):
             cls.owner
         )
 
+    @classmethod
+    def tearDownClass(cls):
+        from django.core.management import call_command
```
Comment (xliiv) [inline src/ralph_assets/tests/unit/tests_other.py]: import to top of file, possible?

**Comment by zefciu** [comment]
:ok:

**Comment by xliiv** [comment]
:ok:

**Pull request merged**